indexer.go

```diff
@@ -48,7 +44,7 @@ func (c CombinedIndexer) Get(ctx context.Context, opts *packages.GetOptions) (pa
 		packages = packages.Join(p)
 	}
 
-	if !opts.Filter.AllVersions {
+	if opts != nil && opts.Filter != nil && !opts.Filter.AllVersions {
```

Comment:
This change was needed after updating this branch with main. 
This Get method is called with opts as `nil` from https://github.com/elastic/package-registry/blob/main/main.go#L290